MTA-6893 Document Hub OIDC authentication (#382)
migtools/mta-documentation: docs/topics/mta-install/proc_change-password-web-console.adoc

:_newdoc-version: 2.18.5
:_template-generated: 2025-07-31
:_mod-docs-content-type: PROCEDURE

[id="change-password-web-console_{context}"]
= Change password in the web console

[role="_abstract"]

> To ensure that your account is not accessible with the default `admin` password, you must change your account password after the first login. When you change your password for the first time, you must also enter your account details.

Reviewer: ```suggestion
To ensure that your account is not accessible with the default `admin` password, change your account password after you log in for the first time. When you change your password for the first time, also specify your account details.
```

> .Procedure
> 
> . Log in to the {ProductShortName} application with `admin` as your password.
> . Click your role dropdown at the top right corner.
> . Select *User account*.
> . Enter the following on the Edit user page:
> * Name
> * Email address
> * A new password in the *Password* field.
> * The same password in the *Confirm Password* field.
> . Click *Save*.

Reviewer: ```suggestion
.Procedure

. Log in to the {ProductShortName} application with `admin` as your password.
. In the upper-right corner, click your role drop-down list.
. Select *User account*.
. On the *Edit user* page, specify the following information:
* Enter your name.
* Enter your email address.
* Enter a new password in the *Password* field.
* Enter the same password in the *Confirm Password* field.
. Click *Save*.
```
Move the location to the beginning of the introductory sentence ("On the Edit user page...") so the user knows where they are before taking actions. 

UI Formatting: Bold the UI page name (Edit user page).

List Introduction: Introduce the list with a complete sentence ending in a colon.

Your bulleted list mixes noun phrases ("Name") with prepositional phrases ("A new password in the..."), all items in a list should be grammatically parallel.

Because these are instructions under a step, every bullet point should start with an imperative verb (like "Enter") and end with a period.
